### ISSUE TITLE
Document a deliver/publish

### ISSUE THREAD

**Issue by thibaultduponchelle** [melezhik/sparky#37]
Is there any plugin to deliver to s3, artifactory, ftp, whatever ?

Could we document it ?

**Comment by melezhik** [comment]
I'd create a Sparrow plugin and use it within sparrowdo scenario to publish to dedicated storage, rather then embedding this feature into Sparky itself. At least this follows Sparky as simple CI server philosophy, where all the complexity and flexibility is expressed in terms of Sparrow plugins. You could see some examples of `aws` plugins - http://sparrowhub.io/search?q=aws , if you need a plugin to publish to s3 bucket for example - let me know and I'll create a plugin for this ...

The same apply for ftp, artfactory and other binary repositories ...

**Comment by thibaultduponchelle** [comment]
Yes, I agree it's on plugin side, this issue is more "show what plugins are available for this and write a cool doc"

**Comment by melezhik** [comment]
Feel free to put here which plugins you need first (s3? artifactory?) and will try to drop a few when I have a time ...
